content/questions/admin/question-088.md

```diff
@@ -0,0 +1,12 @@
+---
+question: "How can you manage access to self-hosted runners in an organization using runner groups?"
+archetype: "questions"
+title: "Question 088"
+draft: false
+---
+
+> https://docs.github.com/en/actions/hosting-your-own-runners/managing-access-to-self-hosted-runners-using-groups
+1. [x] Create runner groups to collect sets of runners, then configure access policies for repositories within the organization.
+1. [ ] Manually assign each self-hosted runner to individual repositories without using runner groups.
+1. [ ] Use a single default runner group for all runners and repositories, with no option to create additional groups or configure access policies.
+1. [ ] Allow unrestricted access to all self-hosted runners from any repository within the organization, without the ability to limit access.
```

Comment:
There is really not much point of having such answers. Wrong answers should be somewhat believable by someone not familiar with the topic.